DEPS

```diff
@@ -54,9 +56,9 @@ hooks = [
   },
   {
     'action': [
-      'bash',
+      'python',
       '-c',
-      'cd src/electron; npm install',
+      'import os; os.chdir("src"); os.chdir("electron"); os.system("npm install")',
```

Comment:
wow
We should probably change in the future.

Reply:
to what?

Reply:
I think it should be a standalone Python script, not a line of code.